PWGJE/EMCALJetTasks/Tracks/AliAnalysisTaskEmcalSoftDropResponse.h

```diff
@@ -64,11 +65,16 @@ class AliAnalysisTaskEmcalSoftDropResponse : public AliAnalysisTaskEmcalJet {
   void SetZcut(double zcut) { fZcut = zcut; }
   void SetReclusterizingAlgorithm(EReclusterizer_t reclust) { fReclusterizer = reclust; }
   void SetSampleFraction(Double_t samplefraction) { fSampleFraction = samplefraction; }
+  void SetMinFractionShared(Double_t minsharedfraction) { fMinFractionShared = minsharedfraction; }
   void SetUseChargedConstituents(bool doUse) { fUseChargedConstituents = doUse; }
   void SetUseNeutralConstituents(bool doUse) { fUseNeutralConstituents = doUse; }
   void SetNameMCParticleContainer(const char *name) { fNameMCParticles = name; }
+  void SetNamePartLevelJetContainer(const char *name) { fPartLevelJetName = name; }
+  void SetNameDetLevelJetContainer(const char *name) { fDetLevelJetName = name; }
+  void SetNameUSLevelJetContainer(const char *name) { fUSLevelJetName = name; }
```

Comment:
Please use meaningful names even if they are longer to spell.  "USLevelJetContainer" is not necessarily clear for everyone what is the meaning, and the task becomes a shared task.